### PR TITLE
feat(web-server): base layout

### LINKED ISSUE #6: Setup empty crates for app and enclave
PRs:

- [ ] #47

### PR DESCRIPTION
**Pull request by PiDelport**
- Closes #6 

### Tasks

- [x] Parameterise signing keys

### CODE REVIEW

**Review by longtomjr — APPROVED** [review]
Looks really good. Thanks a lot!

**Comment by PiDelport** [comment]
@longtomjr: Cool, thanks! There are a few bits of documentation I still want to clean up and add here, then I'll merge this.

File: web-server/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+cargo make --list-all-steps
+```
+
+[cargo make]: https://github.com/sagiegurari/cargo-make
+
+## Rust SGX SDK revision
+
+The codebase is currently pinned to the following revision:
+
+- Commit: [d107bd0718f723221750a4f2973451b386cbf9d2](https://github.com/apache/incubator-teaclave-sgx-sdk/commit/d107bd0718f723221750a4f2973451b386cbf9d2)
```
[the remark this answers is not in the document]
Comment (PiDelport) [reply web-server/README.md:45]: Yeah, this is something we can only document, at this point.

There isn't really a single Cargo.toml we can reference, so I figured the top-level README is the most central place we can document the pinned version at, and update as it changes, along with any related info (like the Intel SGX SDK version, and any other factors to take into account, such as specific fixes or changes we depend on).

[the remark this answers is not in the document]
Comment (PiDelport) [reply web-server/README.md:45]: (Resolving, after checking.)

**Pull request merged**